aldryn_client/cli.py

```diff
@@ -362,30 +366,27 @@ def version(skip_check, show_error):
 
     if not skip_check:
         # check pypi for a newer version
-        try:
-            current_version = StrictVersion(__version__)
-            response = requests.get(
-                'https://pypi.python.org/pypi/aldryn-client/json'
-            )
-            response.raise_for_status()
-            new_version = StrictVersion(response.json()['info']['version'])
+        current_version = StrictVersion(__version__)
+        newest_version, exc = get_latest_version_from_pypi()
 
-            if new_version == current_version:
-                click.echo('\nYou have the latest version of aldryn-client!')
+        if newest_version and newest_version == current_version:
+            click.echo('\nYou have the latest version of aldryn-client!')
 
-            elif new_version > __version__:
-                click.echo(
-                    "\nNew version ({new_version}) available on PyPi. Update "
-                    "now using 'pip install aldryn-client=={new_version}'"
-                    .format(new_version=new_version)
-                )
-        except (requests.RequestException, KeyError, ValueError) as exc:
+        elif newest_version and newest_version > current_version:
+            click.echo(
+                "\nNew version ({newest_version}) available on PyPi. Update "
+                "now using 'pip install aldryn-client=={newest_version}'"
+                .format(newest_version=newest_version)
+            )
+        elif newest_version is False:
             if show_error:
                 click.secho(
-                    '\nThere was an error while trying to retrieve the latest '
-                    'version from pypi.python.org:\n', fg='red'
+                    'There was an error while trying to retrieve the latest '
+                    'version from pypi.python.org',
```

Comment:
The wording could be misunderstood as it actually trying to download the package.